.lintstagedrc

```diff
@@ -0,0 +1,7 @@
+{
+  "*.{js,jsx,tsx}": [
+    "yarn lint",
+    "yarn test:staged",
+    "prettier --write",
+  ]
+}
```

Comment:
Good! Just realised that lint --fix is used in palvelutarjotin, Should we use that also here? For example import ordering can be easily fixed with the current configuration here